src/traffic_ctl/host.cc

```diff
@@ -20,9 +20,22 @@
   See the License for the specific language governing permissions and
   limitations under the License.
  */
+
 #include "traffic_ctl.h"
+#include "HostStatus.h"
 #include <P_RecUtils.h>
-const std::string stat_prefix = "host_status.";
+
+static bool
+verifyReason(char *_reason)
+{
+  for (unsigned int i = 0; i < reasons.size(); i++) {
```

Comment:
```
#include <string_view>
#include <algorithm>
...
std::string_view my_reason { _reason }; // call strlen once, not every loop.
return std::any_of(reasons.begin(), reasons.end(), [my_reason] (std::string const& r) { return r == my_reason; }
```

Reply:
@SolidWallOfCode I refactored this before seeing your comments.  Anyway my changes now, use a char * array encapsulated in a struct.